content/enterprise/admin/infrastructure/active-active.mdx

```diff
@@ -6,6 +6,8 @@ page_title: Active/Active - Infrastructure Administration - Terraform Enterprise
 
 When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](/enterprise/install/automated/active-active) for more information on moving to Active/Active.
 
+~> **Note:** Clustering is no longer supported and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
```

Comment:
It may be worth clarifying that this was the discontinued Kubernetes based Clustering product not just clustering as a concept (I mean that's also not supported but it feels a little ambiguous here if you didn't come from a redirect)

Reply:
Good point! I pushed up a change that is hopefully clearer about what it is that is discontinued and no longer supported. I found similar language on one of the deleted pages, so this seems like a good way to refer to it. @bnferguson Do you feel comfortable approving now?